server/graphql/v1/mutations/collectives.js

```diff
@@ -111,15 +111,23 @@ export async function createCollective(_, args, req) {
   }
 
   const promises = [
-    collective.editTiers(collectiveData.tiers),
     collective.addUserWithRole(req.remoteUser, roles.ADMIN, {
       CreatedByUserId: req.remoteUser.id,
     }),
-    collective.editPaymentMethods(args.collective.paymentMethods, {
-      CreatedByUserId: req.remoteUser.id,
-    }),
   ];
 
+  if (collectiveData.tiers) {
+    promises.push(collective.editTiers(collectiveData.tiers));
+  }
+
+  if (collectiveData.paymentMethods) {
```

Comment:
If I want to delete all the payment methods for the collective, this will return false and it won't work.